README.md

```diff
@@ -568,6 +568,20 @@ which help you to use the various OpenAPI 3 Authentication mechanism.
   will override any default value. This extended property isn't supported in all parts of
   OpenAPI, so please refer to the spec as to where it's allowed. Swagger validation tools will
   flag incorrect usage of this property.
+- `x-go-type-skip-optional-pointer`: specifies if the go type should or should not be a pointer
```

Comment:
```suggestion
- `x-go-type-skip-optional-pointer`: specifies if the Go type should or should not be a pointer
```

Reply:
Thank you for the suggestion @alexandear. Applied.